scripts/rotate_vault.sh

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env bash
+
+init_keepass () {
+    # Locate keepassxc-cli or fail with error
+    if [[ "$OSTYPE" == "darwin"* ]]; then
+        KEE_PASS=/Applications/KeePassXC.app/Contents/MacOS/keepassxc-cli
+    elif [[ "$OSTYPE" == "linux-gnu"* ]]; then
+        KEE_PASS=`which keepassxc-cli`
+    fi
```

Comment:
I'm fine with either. Supporting a configurable `KEE_PASS` env var might be virtuous. I think the scenario that stuck out to me the most was a system that is neither macOS or Linux.  In this scenario, the entire `if` block will be an effective no-op, and any value that happens to exist in the env's `KEE_PASS` var (if one exists) will be attempted.  For 100% safety, I think at least this is warranted:

```bash
else
    echo "unsupported OS: $OSTYPE"
    exit 1
fi

Reply:
With the added user-configurable `KEE_PASS` var now supported (and most importantly, `echo`'d if present), I'm not sure this `else` clause is so important anymore, although it's still more "strict", which I do like (in theory it prevents a situation that might otherwise be confusing to a user). That being said, if someone is trying to run this on OpenBSD/`<insert your Unix variant here>`/etc, I think they'll likely have other bigger fish to fry.